integration-testing/examples/mpp-sample/build.gradle.kts

```diff
@@ -25,6 +25,7 @@ apply(plugin = "kotlinx-atomicfu")
 
 repositories {
     mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/dev")
```

Comment:
Is there a reason to have these repos ordered differently in jvm- and mpp-sample projects?

Reply:
The problem with unresolved classes from `kotlin-stdlib` may've been caused by the maven local repository being corrupted by some other process, and it's priority caused gradle to search for stdlib classes there first. As gradle can not work properly with `mavenLocal`, it's better not to set it first in the repo list .